controllers/api_v1/oauth.go

```diff
@@ -24,11 +24,13 @@ func (oa *OAuthController) generateClient(c *gin.Context) {
 		}
 		c.JSON(errorResponse.Status, errorResponse.NewErrorResponse())
 	} else {
-		dataResponse := &api_helper.DataResponseObject{
-			ID:         oauthClient.ClientID,
-			Type:       "client",
-			Attributes: oauthClient,
+		dataResponse := &api_helper.DataResponse{
```

Comment:
The new naming `DataResponse` not consistent with `ErrorResponseObject` but I understand why there is now `DataResponseObject`, `DataResponse` and `DataResponseArray`.